src/plots.py

```diff
@@ -151,27 +152,43 @@ def plot_histo(
         if layered
         else {}
     )
-
     st.vega_lite_chart(
         data=df,
         spec={
             **CONFIG_MAIN,
             "mark": {"type": "bar", **CONFIG_MARK, "blend": "normal"},
```

Comment:
This will make the plot look a bit nicer 

```suggestion
            "mark": {"type": "bar", **CONFIG_MARK, "blend": "normal", "binSpacing": 0},
```

Reply:
added to the code.